yarn.lock

```diff
@@ -10343,23 +10343,10 @@ eth-ens-namehash@^1.0.2:
     idna-uts46 "^1.0.1"
     js-sha3 "^0.5.7"
 
-eth-hd-keyring@^3.4.0:
-  version "3.4.1"
-  resolved "https://registry.yarnpkg.com/eth-hd-keyring/-/eth-hd-keyring-3.4.1.tgz#c1b34b29bae57d90525dd1b7d3f349a30e0cee8a"
-  integrity sha512-oigRNw9lGRHvhkxZuREV147euMX9GH+0icc/MT0oaU+8hu6KCArg2fm78+uXcIvJvK66DZUS46/+hieXCXqwjw==
-  dependencies:
-    bip39 "^2.2.0"
-    eth-sig-util "^2.4.4"
-    eth-simple-keyring "^3.4.0"
-    ethereumjs-abi "^0.6.5"
-    ethereumjs-util "^5.1.1"
-    ethereumjs-wallet "^0.6.0"
-    events "^1.1.1"
-    xtend "^4.0.1"
-
-"eth-hd-keyring@https://github.com/logvik/eth-hd-keyring":
-  version "3.5.1"
-  resolved "https://github.com/logvik/eth-hd-keyring#0a8ba3351a53e03ad7362714d71287a154f19e47"
+eth-hd-keyring@^3.4.0, eth-hd-keyring@^3.4.1:
```

Comment:
@logvik For future reference, this was the manual lockfile manipulation I was referring to

As long as the version selected is compatible with all of the ranges asked for, you can edit the lockfile to consolidate them. I don't know of any `yarn` command that can do this automatically unfortunately, but it's a neat trick.

Sorry I wasn't able to explain it - I don't really have the words for these sections of the lockfile.